contributing/code/pull_requests.rst

```diff
@@ -147,6 +147,15 @@ work:
     for the ``5.4`` branch, the PR will also be applied by the core team on
     all the ``6.x`` branches that are still maintained.
 
+.. warning::
```

Comment:
isnt "info" enough?
"warning" seem to strong and devs may be not opening new PR :s 

what about:

during a short period of time prior to the release of a new symfony major version, the symfony repository
is feature freeze. Your pull requests are still more than welcome nonetheless they must target the next major version

something like that ^^